presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -68,6 +71,14 @@ public void testGrantRevoke()
         assertThat(() -> bobExecutor.executeQuery(format("DELETE FROM %s WHERE day=3", tableName))).
                 failsWithMessage(format("Access Denied: Cannot delete from table default.%s", tableName));
 
+        // test SHOW GRANTS
```

Comment:
Refactored to have a new test. There are checks to test for SHOW GRANTS ON TABLE x and SHOW GRANTS USER u ON TABLE t in `TestStatementBuilder.java` and `testSqlParser.java`.